package.json

```
@@ -10,24 +10,23 @@
     "benchmark": "concurrently -k -s first \"node ./examples/benchmark/simple.js\" \"autocannon -c 100 -d 30 -p 10 localhost:3000/\"",
     "benchmark:parser": "concurrently -k -s first \"node ./examples/benchmark/parser.js\" \"autocannon -c 100 -d 30 -p 10 -i ./examples/benchmark/body.json -H \"content-type:application/jsoff\" -m POST localhost:3000/\"",
     "build:validation": "node build/build-error-serializer.js && node build/build-validation.js",
-    "coverage": "npm run unit -- --coverage-report=html",
-    "coverage:ci": "tap --coverage-report=html --coverage-report=lcov --allow-incomplete-coverage",
-    "coverage:ci-check-coverage": "tap replay",
+    "coverage": "borp --reporter=./test/test-reporter.mjs --coverage --check-coverage --lines 100",
+    "coverage:ci": "borp --reporter=./test/test-reporter.mjs --coverage --check-coverage",
+    "coverage:ci-check-coverage": "borp --reporter=./test/test-reporter.mjs --coverage --check-coverage --lines 100",
     "lint": "npm run lint:eslint",
     "lint:fix": "eslint --fix",
     "lint:markdown": "markdownlint-cli2",
     "lint:eslint": "eslint",
-    "prepublishOnly": "cross-env PREPUBLISH=true tap --allow-incomplete-coverage test/build/**.test.js && npm run test:validator:integrity",
+    "prepublishOnly": "cross-env PREPUBLISH=true borp --reporter=./test/test-reporter.mjs && npm run test:validator:integrity",
     "test": "npm run lint && npm run unit && npm run test:typescript",
-    "test:ci": "npm run unit -- --coverage-report=lcovonly && npm run test:typescript",
+    "test:ci": "npm run unit && npm run test:typescript",
     "test:report": "npm run lint && npm run unit:report && npm run test:typescript",
     "test:validator:integrity": "npm run build:validation && git diff --quiet --ignore-all-space --ignore-blank-lines --ignore-cr-at-eol lib/error-serializer.js && git diff --quiet --ignore-all-space --ignore-blank-lines --ignore-cr-at-eol lib/configValidator.js",
     "test:typescript": "tsc test/types/import.ts --noEmit && tsd",
     "test:watch": "npm run unit -- --watch --coverage-report=none --reporter=terse",
-    "unit": "tap",
-    "unit:junit": "tap-mocha-reporter xunit < out.tap > test/junit-testresults.xml",
-    "unit:report": "tap --coverage-report=html --coverage-report=cobertura | tee out.tap",
-    "citgm": "tap --jobs=1 --timeout=120"
+    "unit": "borp --reporter=./test/test-reporter.mjs --coverage --check-coverage",
+    "unit:report": "borp --reporter=./test/test-reporter.mjs --coverage > out.test-report",
```

This is not equivalent. It should be:
```suggestion
    "unit:report": "c8 --reporter html borp --reporter=./test/test-reporter.mjs",
```